src/lib/utils/audio.ts

```diff
@@ -163,7 +165,7 @@ export const cleanSongName = (songName: string, artistNames: string[]): string[]
 
 	// Remove any strings that contain an artist as this isn't the song name.
 	const songNamesNoArtist = [songWithoutSuffixAndPrefix, songWithoutPrefix, songWithoutSuffix].filter(
-		(str) => !artistNames.some((artist) => str.includes(artist))
+		(songName) => !artistNames.some((artist) => jaroWinkler(songName, artist) >= kGuessThreshold)
 	);
 
 	const validSongVariations = [...new Set([...songNamesNoArtist, normalized])];
```

Comment:
Something I just realized is that the `.map(() => convertToAlphaNumeric())` below this should probably be done before the names are converted to a set in back in case one name has like an extra space or something, but after it becomes alphanumeric, it's the same as another.